bin/find-dev-deps.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const json = fs.readFileSync('package.json', { encoding: 'utf-8' });
+const meta = JSON.parse(json);
+const deps = Object.assign({}, meta.dependencies, meta.peerDependencies);
+
+Object.keys(deps).forEach((key) => {
+    if (deps[key].indexOf('dev') !== -1) {
+        process.exit(-1);
```

Comment:
Not sure if `-1` is a valid exit status, as they are [capped to mod 255 in bash](http://tldp.org/LDP/abs/html/exitcodes.html), and [unsigned on Windows](https://en.wikipedia.org/wiki/Exit_status#Windows). I am wondering if that has implications, though.

![image](https://user-images.githubusercontent.com/90405/28624132-cd436838-7220-11e7-87bd-054cacf1a2ae.png)